lib/improver/cli/blend_adjacent_points.py

```diff
@@ -82,13 +82,10 @@ def main(argv=None):
                         'dimension: ie when all files provided are from the '
                         'same forecast cycle.')
     parser.add_argument('weighting_mode', metavar='WEIGHTED_BLEND_MODE',
-                        choices=['weighted_mean', 'weighted_maximum'],
+                        choices=['weighted_mean'],
```

Comment:
You need to update the description for this CLI too.

Reply:
Well spotted - thanks!